packages/fxa-auth-server/lib/routes/emails.js

```diff
@@ -1036,6 +1037,39 @@ module.exports = (
           uid,
         });
 
+        return {};
+      },
+    },
+    {
+      method: 'POST',
+      path: '/emails/reminders/cad',
+      options: {
+        auth: {
+          strategy: 'sessionToken',
+        },
+      },
+      handler: async function (request) {
+        log.begin('Account.CadReminderEmail', request);
+
+        const sessionToken = request.auth.credentials;
+        const uid = sessionToken.uid;
+
+        const reminders = await cadReminders.get(uid);
+        const exists = cadReminders.keys.some((key) => {
+          return reminders[key] !== null;
+        });
+
+        if (!exists) {
+          await cadReminders.create(uid);
```

Comment:
I assume this create method handles the type of interval it's creating internally.